rails_programming/rails_basics/deployment.md

```diff
@@ -143,3 +153,13 @@ This section contains helpful links to other content. It isn't required, so cons
 * [Heroku Custom Domains](https://devcenter.heroku.com/articles/custom-domains) help file
 * [Converting from PostgreSQL to SQLite3](http://manuel.manuelles.nl/blog/2012/01/18/convert-postgresql-to-sqlite/)
 * [Webinar on Using Heroku in Production](https://blog.heroku.com/archives/2013/7/11/running-production-apps-on-heroku)
+
+### Knowledge Check
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+* <a class="knowledge-check-link" href="#dyno-knowledge-check">What are dynos? How many of them are required to run your application?</a>
+* <a class="knowledge-check-link" href="#domain-knowledge-check">What are the steps required in order to change my application's domain name to `https://the-cool-app.com`?</a>
```

Comment:
Small typo. it should be "https://the-cool-app.com?". If we could change "my" to "your" as well that would be awesome!

Reply:
Not sure where's the actual typo. Have looked several times now but can't figure out. Can you be more specific?